composer.json

```diff
@@ -81,18 +81,18 @@
 			"@ci:composer:require-checker"
 		],
 		"ci:composer:normalize": "@composer normalize --dry-run",
-		"ci:composer:require-checker": "composer-require-checker",
+		"ci:composer:require-checker": "@php composer-require-checker",
```

Comment:
You can also directly use `@composer` like this: https://github.com/TYPO3-Documentation/tea/blob/main/composer.json#L106

Reply:
Ah, sorry, I misread the change. Then my proposal does not apply.

Reply:
Yeah, it's a dedicated binary ;) @composer is already in place where possible.